database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -17,7 +17,9 @@ public function up()
             $table->increments('id');
             $table->string('name');
             $table->string('email')->unique();
+            $table->boolean('verified')->default(false);
             $table->string('password');
+            $table->softDeletes();
```

Comment:
We need these changes in a new migration file.

Reply:
no need , we don't have release or something on production .

Reply:
We already do. 😁 

Can you at least move these changes to `database/migrations/2018_06_19_025133_create_user_verification_tokens_table.php`?